src/components/Callout/Callout.md

```diff
@@ -143,37 +143,43 @@ const HintedPositionCallout = (props) => (
 );
 
 <div style={{ height: '400px', display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
-  <FixedGridRow>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.topRightEdge} align="left" label="Top right" />
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.topCenter} align="center" label="Top center" />
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.topLeftEdge} align="right" label="Top left" />
-    </FixedGridColumn>
-  </FixedGridRow>
-  <FixedGridRow>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.leftCenter} align="left" label="Left center" />
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.rightCenter} align="right" label="Right center" />
-    </FixedGridColumn>
-  </FixedGridRow>
-  <FixedGridRow>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.bottomRightEdge} align="left" label="Bottom right" />
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.bottomCenter} align="center" label="Bottom center" />
-    </FixedGridColumn>
-    <FixedGridColumn fixed={true} width={100}>
-      <HintedPositionCallout hint={DirectionalHint.bottomLeftEdge} align="right" label="Bottom left" />
-    </FixedGridColumn>
-  </FixedGridRow>
+  <div>
+    <FixedGridRow>
+      <FixedGridColumn fixed={true} width={100}>
+        <HintedPositionCallout hint={DirectionalHint.topRightEdge} align="left" label="Top right" />
+      </FixedGridColumn>
+      <FixedGridColumn fixed={true} width={100}>
+        <HintedPositionCallout hint={DirectionalHint.topCenter} align="center" label="Top center" />
+      </FixedGridColumn>
+      <FixedGridColumn fixed={true} width={100}>
+        <HintedPositionCallout hint={DirectionalHint.topLeftEdge} align="right" label="Top left" />
+      </FixedGridColumn>
+    </FixedGridRow>
+  </div>
+  <div>
+    <FixedGridRow>
+      <FixedGridColumn fixed={true} width={100}>
+        <HintedPositionCallout hint={DirectionalHint.leftCenter} align="left" label="Left center" />
+      </FixedGridColumn>
+      <FixedGridColumn fixed={true} width={100}>
+      </FixedGridColumn>
+      <FixedGridColumn fixed={true} width={100}>
+        <HintedPositionCallout hint={DirectionalHint.rightCenter} align="right" label="Right center" />
+      </FixedGridColumn>
+    </FixedGridRow>
+  </div>
+  <div>
```

Comment:
Why are the extra `div`s necessary?

Reply:
took a little head-scratching to figure out why the visual diffs were messed up for this story. basically i'd fixed a FixedGridRow bug via `flex-basis: 100%` because previously it would only fill its horizontal space if its column content pushed it out that far. in your example, your flex container's direct children were also display: flex and now have flex-basis: 100% so they filled the entire vertical space instead of being clustered in the center of the wrapper.

in general it's best to not use flexbox containers as direct children of another flexbox container, you should have another div or something to manage the flex-child layout then nest the next flex container under that

Reply:
Aah I didn't stop to think that `FixedGridRow` was also `display: flex`.